### DISCUSSION TITLE
Phone Meeting Info

### DISCUSSION THREAD

**Discussion by ATLCOTech** [code4recovery/12-step-meeting-list#1233] [Q&A]
Hi - we noticed that the meetings that have only phone meetings are not listed on  the  meeting guide app. I'm thinking that the information is in the wrong place.  I've attached an example of 1 meeting this is occurring.
[meeting error.xlsx](https://github.com/code4recovery/tsml-ui/files/12898460/meeting.error.xlsx)

**Comment by joshreisner** [comment]
hi @ATLCOTech , to clarify, when you say the "meeting guide app" do you mean it shows on your website but not on the blue chair mobile app?

is this the meeting on your website? https://atlantaaa.org/meetings/new-beginnings-womens-group-tara-club-virtual-sat-1200pm/ the information in the spreadsheet is slightly different than what i'm seeing at that URL

**Comment by ATLCOTech** [comment]
Hi,

Thanks for getting back to quickly. Yes the info doesn’t show on the
meeting guide app. I’m not sure if I’ve got the info in the wrong place.
Meeting guide screenshot is attached.

Mark Just
678-368-8368

**Comment by joshreisner** [comment]
hey mark! yeah so the spreadsheet has info in the `conference_url` field which is great, that's what you need. but the meeting i linked to on atlantaaa.org above does not appear to. i see a phone number in the group_notes field, you want that in conference URL so it will enable the join meeting button. 

also, we recommend not using the office address for online meetings, rather use an approximate location such as `Atlanta, GA, USA`

if you make those changes and then wait a few hours, you should start seeing the meeting show up in the app

**Comment by ATLCOTech** [comment]
Hey there,

That is correct. I double checked the info on the site against the
spreadsheet and it looked the same to me. What did you see that was
different?
Sincerely,

Mark J
Tech & Newsletter Chair

If you have a newsletter submission, please send to ***@***.***
.
If you have a website comment, issue or would like to join the tech/web
committee email ***@***.*** ***@***.***>
To contact the steering committee email ***@***.***
Central Office ***@***.***

**Comment by joshreisner** [comment]
hi @ATLCOTech , i've moved this to the 12 Step Meeting List discussions area, since this is a question about the WordPress plugin.

what version of the WordPress plugin are you using? it should say on the plugins page:

<img width="779" alt="Screenshot 2023-10-14 at 9 58 25 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/6a525872-3546-48b6-9a60-cdf031f20810">

i think you may be using either an old and/or very customized version of the plugin, but just want to confirm first

**Reply by joshreisner** [reply]
hey @ATLCOTech i took a look at atlantaaa.org's JSON feed that the Meeting Guide app uses, it's not providing any online meeting information at all -- @onemooresam would you have any context on the history with this site and online meetings?

**Reply by onemooresam** [reply]
In September, the site's custom code for TSML plugin was overwritten when someone updated the plugin. The site was restored from a Digital Ocean backup on 20SEP. That *may* have interrupted the feed. There is a dev site (https://averym8.sg-host.com/) that started from a clone of the live site. TSML is not set to feed to any aggregator there.

**Reply by onemooresam** [reply]
On the dev site, the custom code was removed and TSML plugin is current. The meeting information is cleaned and correctly formatted for the current TSML.

**Reply by joshreisner** [reply]
great, thanks @onemooresam ! 

@ATLCOTech i think this is your answer ☝️ - there was a restore done last month to the live site which is causing the Meeting Guide app not to get the right info for online meetings. 

promoting the dev site code & data to the production site should fix it
